ldes-server-port-fetch-rest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/treenode/services/EventStreamInfoResponseTest.java

```diff
@@ -0,0 +1,49 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.rest.treenode.services;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFParser;
+import org.apache.jena.vocabulary.RDF;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static be.vlaanderen.informatievlaanderen.ldes.server.domain.constants.RdfConstants.*;
+import static org.apache.jena.rdf.model.ResourceFactory.createResource;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class EventStreamInfoResponseTest {
+
+    @Test
+    void convertToStatements_IncludesMandatoryStatements() {
+        String eventStreamId = "http://example.com/eventStream";
+        EventStreamInfoResponse response =
+                new EventStreamInfoResponse(eventStreamId, "", "", null, List.of());
+
+        List<Statement> statements = response.convertToStatements();
+
+        assertThat(statements).anyMatch(s ->
+                s.getSubject().toString().equals(eventStreamId) &&
+                        s.getPredicate().equals(RDF_SYNTAX_TYPE) &&
+                        s.getObject().toString().equals(LDES_EVENT_STREAM_URI));
+    }
+
+    @Test
+    void convertToStatements_AddsShapeIfPresent() {
+        Model shape = RDFParser.fromString("[ a <http://www.w3.org/ns/shacl#NodeShape> ]").lang(Lang.TURTLE).toModel();
+
+        EventStreamInfoResponse response =
+                new EventStreamInfoResponse("http://example.com/eventStream", "", "", shape, List.of());
+
+        List<Statement> statements = response.convertToStatements();
+
+        Model model = ModelFactory.createDefaultModel().add(statements);
+        Resource shapeSubject = model.listSubjectsWithProperty(RDF.type, createResource(NODE_SHAPE_TYPE)).nextResource();
+        assertThat(model.listObjectsOfProperty(TREE_SHAPE).hasNext()).isTrue();
```

Comment:
`assertThat(iterator).hasNext()` also exits, resulting in not having to chain the assertion with `.isTrue()`